sdv/multi_table/base.py

```diff
@@ -429,33 +429,32 @@ def get_constraints(self):
 
         return constraints
 
-    def load_custom_constraint_classes(self, table_name, filepath, class_names):
+    def load_custom_constraint_classes(self, filepath, class_names):
         """Load a custom constraint class for the specified table's synthesizer.
 
         Args:
-            table_name (str):
-                Table to add constraint to.
             filepath (str):
                 String representing the absolute or relative path to the python file where
                 the custom constraints are declared.
             class_names (list):
                 A list of custom constraint classes to be imported.
         """
-        self._table_synthesizers[table_name].load_custom_constraint_classes(filepath, class_names)
+        for t_name in self._table_synthesizers:
```

Comment:
minor: you can loop through the synthesizers directly
```python
for synthesizer in self._table_synthesizers.values():
    ...
```
Also if not too long, can we use full length names (ie. `table_name` instead of `t_name`)